core/test/database/sqlcipher_tests.cpp

```diff
@@ -63,16 +64,6 @@ TEST(SQLCipherTest, SanityCheck) {
                 soci::use(value_0), soci::use(value_1);
         session.close();
     }
-    {
```

Comment:
I don’t get why you removed that from the test suite.

Reply:
It was not removed it was moved to separate test that can only be run on not broke compilers. Will elaborate inside the next comment.